store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10357,4 +10357,9 @@ TODO: delete them permanently from here
   <globalConfigValue>TRUE</globalConfigValue>
   <desc>whether to enable CountObjectsRequest with userAccount/account type on Zimbra Admin Console</desc>
 </attr>
+
+<attr id="4098" name="zimbraFeatureAllowUsernameInPassword" type="boolean" cardinality="single" optionalIn="account,cos,domain" flags="accountInfo,accountCosDomainInherited" since="10.1.0">
```

Comment:
a question related to requirements, should we remove LC value now and add some migration step to move LC value to ldap attribute so customers don't loose data in migration/upgrade

Reply:
Saket, under this PR we have just introduced the placeholder for LDAP attribute so it can be shipped to next release. the implementation will be done as a part of ZCS-13218, where in we should take care of the above case you mentioned.

Reply:
Thanks